dropwizard-migrations/src/main/java/io/dropwizard/migrations/AbstractLiquibaseCommand.java

```diff
@@ -72,16 +75,21 @@ private CloseableLiquibase openLiquibase(final PooledDataSourceFactory dataSourc
             throws SQLException, LiquibaseException {
         final CloseableLiquibase liquibase;
         final ManagedDataSource dataSource = dataSourceFactory.build(new MetricRegistry(), "liquibase");
-
+        final Database database = createDatabase(dataSource, namespace);
         final String migrationsFile = namespace.getString("migrations-file");
         if (migrationsFile == null) {
-            liquibase = new CloseableLiquibaseWithClassPathMigrationsFile(dataSource, migrationsFileName) {
-            };
+            liquibase = new CloseableLiquibaseWithClassPathMigrationsFile(dataSource, database, migrationsFileName);
         } else {
-            liquibase = new CloseableLiquibaseWithFileSystemMigrationsFile(dataSource, migrationsFile);
+            liquibase = new CloseableLiquibaseWithFileSystemMigrationsFile(dataSource, database, migrationsFile);
         }
 
-        final Database database = liquibase.getDatabase();
+        return liquibase;
+    }
+
+    private Database createDatabase(final ManagedDataSource dataSource, Namespace namespace) throws SQLException, LiquibaseException {
+        DatabaseConnection conn = new JdbcConnection(dataSource.getConnection());
```

Comment:
Could you please make the `conn` and `database` variables `final`? This will be consistent with rest of the codebase.